Baconit/Strings/es-es/Resources.resw

```diff
@@ -0,0 +1,889 @@
+  </data>
+  <data name="DisablePostLoad.Text" xml:space="preserve">
+    <value>No se carga el contenido hasta que se pulse la pantalla.</value>
+  </data>
+  <data name="EverythingCB.Content" xml:space="preserve">
+    <value>Todo</value>
+  </data>
+  <data name="FatalException.Text" xml:space="preserve">
+    <value>No dejes que las excepciones fatales maten la app, informa de las excepciones en lugar de romper.(Esto dejará la app en un estado inestable)</value>
+  </data>
+  <data name="FlipViewS.Text" xml:space="preserve">
+    <value>Configuración de la vista</value>
+  </data>
+  <data name="fullpost.Text" xml:space="preserve">
+    <value>Muestra todos los títulos del post.</value>
+  </data>
+  <data name="HeaderAboutTB.Text" xml:space="preserve">
+    <value>Sobre</value>
+  </data>
+  <data name="HowMessagesTB.Text" xml:space="preserve">
+    <value>Como los mensajes aparecerán en el centro de notificaciones.</value>
+  </data>
+  <data name="ImageUpdateFTB.Text" xml:space="preserve">
+    <value>Frecuencia de actualización de la imagen.</value>
+  </data>
+  <data name="IsNNSFW.Content" xml:space="preserve">
+    <value>No</value>
+  </data>
+  <data name="IsNSFW.Content" xml:space="preserve">
+    <value>¿Es contenido para adultros?</value>
+  </data>
+  <data name="IsYNSFW.Content" xml:space="preserve">
+    <value>Sí</value>
+  </data>
+  <data name="LastUpdate.Text" xml:space="preserve">
+    <value>Última actualización:</value>
+  </data>
+  <data name="LoadingC.Text" xml:space="preserve">
+    <value>Cargando comentarios.</value>
+  </data>
+  <data name="LockScreenTB.Text" xml:space="preserve">
+    <value>Origne de la imagen de la pantalla de bloqueo.</value>
+  </data>
+  <data name="LoginCT.Content" xml:space="preserve">
+    <value>Entrar</value>
+  </data>
+  <data name="MemoryPressure.Text" xml:space="preserve">
+    <value>Las páginas se limpiaron para guardar memorio: 0</value>
+  </data>
+  <data name="MemOverlay.Text" xml:space="preserve">
+    <value>Muestra el consumo de memoria:</value>
+  </data>
+  <data name="MSBand1.Text" xml:space="preserve">
+    <value>Reddunt puede integrarse con tu Microsoft Band v1 o v2 para actualizar tu fondo de pantalla,añadir el fondo de Reddunt o darte notificaciones instantáneas de tus mensajes entrantes.</value>
+  </data>
+  <data name="MSBand2.Text" xml:space="preserve">
+    <value>No se encontró ninguna Band, comprueba que tu Band está conectada y con batería.</value>
+  </data>
+  <data name="MSBand3.Text" xml:space="preserve">
+    <value>Fondo de pantalla</value>
+  </data>
+  <data name="MSBand4.Text" xml:space="preserve">
+    <value>Automáticamente actualiza el fondo de pantalla de Band</value>
+  </data>
+  <data name="MSBand5.Text" xml:space="preserve">
+    <value>Origen del fondo de pantalla de Band</value>
+  </data>
+  <data name="MSBand6.Text" xml:space="preserve">
+    <value>Azulejo &amp; Entrada</value>
```

Comment:
Another word that is nos commonly used to refer the tiles in this region.
As well with spoiler we use the untranslated words. [Here's an example](https://hipertextual.com/archivo/2013/10/personalizar-las-live-tiles-de-windows-8/) and [here's another](https://msdn.microsoft.com/es-es/library/dn151515.aspx). They are older articles, but they are useful for exposing my point. The second article is from the es-es Microsoft translation.